src/util/wallet-utils/src/callContract.ts

```diff
@@ -47,5 +55,25 @@ export const callContractFordefi: CallContract = async (params) => {
 };
 
 export const callContractAsigna: CallContract = async (params) => {
-  throw new Error("Not implemented");
+  const tx = deserializeTransaction(params.txHex);
+  const payload = tx.payload as ContractCallPayload;
+
+  return new Promise<string>((resolve, reject) =>
+    openContractCall(
```

Comment:
If I'm reading the docs correctly, this will alwsa submit the tx (https://www.npmjs.com/package/@stacks/connect/v/7.10.1-alpha.f3096a1.0#calling-smart-contracts-opencontractcall):
> Calling Smart-Contracts (openContractCall)
> Calling smart-contracts lets users interact with the blockchain through transactions.
>
> The snippet below will open the wallet to confirm and broadcast a smart-contract transaction.

Reply:
matteo you're a genius

Reply:
Oh right, then I guess we need to change `src/app/withdraw/components/withdraw-client.tsx` to skip the broadcast for asigna, as well as considering the case of tx not existing in `getWithdrawalInfo`

Reply:
exactly my thinking

Reply:
so unfortunately interaction with the wallet atm is very limited will keep working on it but for the time being the minimum is met i think which is get the tx out starting the process, the idea tho is in the status, which can be polled by using the initiate tx id /withdraw/[txid] we need to somehow to go from withdraw -> asigna sign and execute -> status the flow breaks in the middle step